invenio_records_lom/resources/serializers/__init__.py

```diff
@@ -106,13 +121,15 @@ def build_location(self, jsn, parent_tag):
         """Append location to parent_tag."""
         tag = self.element_maker.location(
             jsn["#text"],
-            **{"{http://www.w3.org/XML/1998/namespace}lang": jsn["type"]},
+            # while some of LOM-UIBK's examples include a type-attribute here, most don't
```

Comment:
if you talk about the LOM-UIBK here, then you should mention it somewhere with a link. maybe in the README?

Reply:
done